Compiler/FrontEnd/ExpressionSimplify.mo

```diff
@@ -4777,6 +4777,11 @@ algorithm
       Real r;
       Option<DAE.Exp> oexp;
 
+    // fix for PNLib where "{} and not {}" is evaluated
+    case (_,_,DAE.ARRAY(array={}),_)
+      then origExp;
+    case (_,_,_,DAE.ARRAY(array={}))
+      then origExp;
```

Comment:
@sjoelund Would it be okay to return the empty arrays in these two cases (instead of `origExp`)?
This would reduce the `{} and not {}` of PNLib to `{}`
I could add these cases again, if that is okay. I don't know if these rules would shadow some error or so.